src/routes/IframeCardPage.tsx

```diff
@@ -18,11 +20,14 @@ import {
   MixpanelEventType,
   MixpanelPaymentPhase,
 } from "../utils/mixpanel/mixpanelEvents";
+import InformationModal from "../components/modals/InformationModal";
 
 export default function IFrameCardPage() {
   const navigate = useNavigate();
   const [loading] = React.useState(false);
   const [hideCancelButton, setHideCancelButton] = React.useState(false);
+  const [modalOpen, setModalOpen] = React.useState(false);
```

Comment:
Maybe it should be renamed to something that makes it clear the modal refers to the CVV?